scripts/generators/patternfly-4-component/templates/component/{{componentName}}.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
+import styles from '@patternfly/patternfly-next/{{typeDir}}/{{componentName}}/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
 
-const propTypes = {};
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string
+};
 
-const defaultProps = {};
+const defaultProps = {
+  children: null,
+  className: ''
+};
 
-const {{componentName}} = () => <div />;
+const {{componentName}} = ({ className, children, ...props }) => (
+  <div {...props} className={css(styles, className)}>
```

Comment:
Styles will never be used here. Maybe make this styles.{{camelcase componentName}} the build will atleast fail if it is incorrect.